video/out/opengl/ra_gl.c

```diff
@@ -2,6 +2,9 @@
 
 #include "formats.h"
 #include "utils.h"
+#if defined(_WIN32) && (defined(__aarch64__) || defined(__arm64ec__) || defined(__arm__))
+#undef MemoryBarrier
```

Comment:
Probably would be better to rename the function pointer name in mpv to avoid this conflict.